enterprise/server/util/oci/oci.go

```diff
@@ -24,8 +24,9 @@ import (
 )
 
 var (
-	registries = flag.Slice("executor.container_registries", []Registry{}, "")
-	mirrors    = flag.Slice("executor.container_registry_mirrors", []MirrorConfig{}, "")
+	registries         = flag.Slice("executor.container_registries", []Registry{}, "")
+	mirrors            = flag.Slice("executor.container_registry_mirrors", []MirrorConfig{}, "")
+	useDefaultKeychain = flag.Bool("executor.container_registry_default_keychain_enabled", false, "Enable the default container registry keychain, respecting both docker configs and podman configs.")
```

Comment:
Could we call this defaultKeychainEnabled for consistency / clarity?

Reply:
woops, renamed the flag and forgot to rename the var. Done